src/routes/Project/containers/TableContainer.js

```diff
@@ -2,14 +2,21 @@ import { connect } from 'react-redux';
 
 import Table from '../components/Table';
 import { updateClip } from '../actions/TrackActions';
+import { resizeTable } from '../actions/TableActions';
 
 const mapDispatchToProps = {
-  updateClip
+  updateClip,
+  resizeTable
 };
 
 const mapStateToProps = (state) => ({
   numTracks: Object.keys(state.tracks).length,
-  tracks: state.tracks
+  tracks: state.tracks,
+  widthPx: state.table.widthPx,
+  timeInterval: state.table.timeInterval,
```

Comment:
I was thinking how many of these actually need to be part of the redux state, as in how many of these should actually be persistent and not reset when you open the project again. If they can be reset I would move them to the component state.